extensions/positron-assistant/src/config.ts

```diff
@@ -144,6 +144,22 @@ export async function showConfigurationDialog(context: vscode.ExtensionContext,
 		.filter((source) => {
 			// If no specific set of providers was specified, include all
 			return enabledProviders.length === 0 || enabledProviders.includes(source.provider.id);
+		})
+		.map((source) => {
+			// Resolve environment variables in apiKeyEnvVar
+			if ((source.defaults as any).apiKeyEnvVar) {
+				const envVarName = (source.defaults as any).apiKeyEnvVar.key;
```

Comment:
Remove cast to any for strongly-typed alternative:
```suggestion
			if ('apiKeyEnvVar' in source.defaults && source.defaults.apiKeyEnvVar) {
				const envVarName = source.defaults.apiKeyEnvVar.key;
```